### DISCUSSION TITLE
How acceptedFileTypes should work?

### DISCUSSION THREAD

**Discussion by lotarbo** [awcodes/filament-curator#156] [Q&A]
Hi. i have field with `acceptedFileTypes(['image/jpeg', 'image/png'])` and I can't upload any other file, but I can pick any file from already uploaded. I can pick mp4 video for this field. 
So i don't understand it's a bug or I should add additional rules like `rules(['image'])` ?)

**Comment by awcodes** [comment]
Sounds like a bug. Can you report it as an issue please and I'll look into it as soon as I can. Thanks.
